### ISSUE TITLE
Feature request: Calibre conversion plugin

### ISSUE THREAD

**Issue by d-faure** [pgaskin/kepubify#68]
I would love to get a Calibre integrated alternative to KTE (even if I know this is more a Calibre than a kepubify improvement request).

**Comment by pgaskin** [comment]
This is a duplicate of #17, but the situation has changed a bit, so I'm willing to reconsider this. With the rewrite of the kepubify conversion code I did for v4, this would be much easier to implement than before. Also, I think I have a better understanding of the architecture of Calibre plugins now compared to back then.

I'll experiment with this when I have time.

**Comment by d-faure** [comment]
Nice!
IMHO, it should even be possible to get a conversion plugin acting "simply" as a shell for the kepubify binary, piping in and out the source epub content to the converted to kepub data and bought back to calibre.

**Comment by malfroid** [comment]
Agreed, it would be great to have a kepubify plugin for Calibre. The plugin could point to an existing `kepubify` binary file path via the plugin configuration. 

The Calibre docs have improved over the years:

- https://manual.calibre-ebook.com/creating_plugins.html
- https://manual.calibre-ebook.com/conversion.html

**Comment by pgaskin** [comment]
Not currently planning to implement this myself, since:

- I have no interest in Calibre.
- Calibre conversion code has improved recently.
- It's possible to use kepubify on Calibre libraries manually.
- Quite a few other tools (e.g., calibre-web, komga) support calling kepubify themselves.
